mention-storer/pom.xml

```diff
@@ -20,23 +20,23 @@
         </dependency>
     </dependencies>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-maven-plugin</artifactId>
-          <configuration>
-              <mainClass>de.brandwatch.minianalytics.mentionstorer.MentionStorerApplication</mainClass>
-          </configuration>
-          <executions>
-              <execution>
-                  <goals>
-                      <goal>repackage</goal>
-                  </goals>
-              </execution>
-          </executions>
-      </plugin>
-    </plugins>
-  </build>
+    <build>
```

Comment:
How did this diff happen? Either it was not formatted correctly before or now?

Reply:
Before